docs/stdlib/cfg.rst

```diff
@@ -85,6 +85,9 @@ Query planning
 Client connections
 ------------------
 
+:eql:synopsis:`allow_user_specified_id -> bool`
+  Makes it possible to set the `.id` property when inserting new objects.
```

Comment:
We should add a warning about the security implications of this and that custom object ids should come from a trusted source.

Reply:
added